### PR TITLE
Cnnappnews 247

### PR DESCRIPTION
**Pull request by dsover**
CNNAPPNEWS-247

This PR fixes issue where older videos have bad variances associated to them. if the video is older than a specific age, we need to remove the associated variance from the video url.

### CODE REVIEW

File: lib/hlsConvertor.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        };
+
+    /**
+     * used to remove bad variances from video urls. older videos do not have as many options
+     *
+     * @param {Object} videoObject
+     * Video object model contains.
+     * @param {String} _
+     * this is the video url
+     * @param {String} $
```
Comment (khowen) [inline lib/hlsConvertor.js]: Would we classify "_" this as a param for the function? Also, I don't see $ used anywhere in this function.

File: test/mocks/nocks.js [hunk 2]
```diff
@@ -213,7 +213,7 @@ nock('http://data.cnn.com:80', {"encodedQueryParams":true})
 
 // #21
 nock('http://hypatia.api.cnn.com:80', {"encodedQueryParams":true})
-    .get('/svc/content/v2/search/collection1/type:gallery/dataSource:api.greatbigstory.com/rows:10/sort:lastPublishDate')
+    .get('/svc/content/v2/search/collection1/type:gallery/dataSource:api.greatbigstory.com/rows:10/sort:lastModifiedDate')
```
Comment (khowen) [inline test/mocks/nocks.js]: `npm test` doesn't pass for me. Does it for you? Changing this here will require re-recording the nocks.

File: lib/hlsConvertor.js (hunk 1 above)
Comment (dsover) [inline lib/hlsConvertor.js]: removed these, they're no longer used

File: test/mocks/nocks.js (hunk 2 above)
Comment (dsover) [inline test/mocks/nocks.js]: Yeah, test works for me. Here is the travis ci build link 
https://travis-ci.org/cnnlabs/cnn-content-retriever/builds/238009816?utm_source=github_status&utm_medium=notification

**Pull request merged**